Change defaults for hash_max and elasticity. Fix to initialize the values when mdb is created too.

diff --git a/net/bridge/br_multicast.c b/net/bridge/br_multicast.c
index 4feb18a..ed5d394 100644
--- a/net/bridge/br_multicast.c
+++ b/net/bridge/br_multicast.c
@@ -593,7 +593,7 @@ struct net_bridge_mdb_entry *br_multicast_new_group(struct net_bridge *br,
 
 	mdb = rcu_dereference_protected(br->mdb, 1);
 	if (!mdb) {
-		err = br_mdb_rehash(&br->mdb, BR_HASH_SIZE, 0);
+		err = br_mdb_rehash(&br->mdb, br->hash_max, br->hash_elasticity);
 		if (err)
 			return ERR_PTR(err);
 		goto rehash;
@@ -1657,8 +1657,8 @@ static void br_multicast_query_expired(unsigned long data)
 
 void br_multicast_init(struct net_bridge *br)
 {
-	br->hash_elasticity = 4;
-	br->hash_max = 512;
+	br->hash_elasticity = 4096;
+	br->hash_max = 4096;
 
 	br->multicast_router = 1;
 	br->multicast_querier = 0;
